SHOULDERS.md

```diff
@@ -1,12 +1,556 @@
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/credentials](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/credentials)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/encoding](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/encoding)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/encoding/proto](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/encoding/proto)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/grpclog](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/grpclog)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/internal](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/internal)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/internal/backoff](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/internal/backoff)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/internal/channelz](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/internal/channelz)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/internal/grpcrand](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/internal/grpcrand)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/keepalive](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/keepalive)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/metadata](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/metadata)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/naming](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/naming)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/peer](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/peer)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/resolver](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/resolver)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/resolver/dns](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/resolver/dns)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/resolver/passthrough](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/resolver/passthrough)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/stats](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/stats)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/status](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/status)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/tap](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/tap)
+
+* [github.com/gomods/athens/vendor/google.golang.org/grpc/transport](https://godoc.org/github.com/gomods/athens/vendor/google.golang.org/grpc/transport)
```

Comment:
@mydiemho do you mind removing the dependencies from the `vendor` directory?

Reply:
As well I don't think our sub packages should be there either.